src/components/fields/SchemaField.js

```diff
@@ -91,7 +91,7 @@ function DefaultTemplate(props) {
   }
 
   return (
-    <div className={classNames}>
+    <div id={id} className={classNames}>
```

Comment:
yes. it would. this is a just a naive implementation so we can unpack what needs to be done.